tardis/plasma/properties/property_collections.py

```diff
@@ -1,5 +1,17 @@
+from tardis.plasma.properties import (BetaRadiation, LevelBoltzmannFactor,
```

Comment:
I think you want to take this out here. That was already changed with your last PR.